packages/hash/frontend/src/blocks/page/createSuggester/fuzzySearchBy.ts

```diff
@@ -1,25 +1,15 @@
-/**
- * used to find matches of search in the given choices using the fuzzy search algorithm
- * @see https://en.wikipedia.org/wiki/Approximate_string_matching
- */
 export const fuzzySearchBy = <T>(
   choices: T[],
   search: string,
   getter: (choice: T) => string,
 ): T[] => {
-  const lowerSearch = search.toLowerCase();
-  const searchLength = search.length;
-
   return choices.filter((choice) => {
     const match = getter(choice).toLowerCase();
 
-    let offset = 0;
-    for (let searchIndex = 0; searchIndex < searchLength; searchIndex++) {
-      const matchIndex = match.indexOf(lowerSearch.charAt(searchIndex), offset);
-      if (matchIndex < 0) return false;
-      offset = matchIndex + 1;
+    if (match.includes(search)) {
```

Comment:
The old fuzzySearch function was too loose in its strictness, as seen in the demo. I've chosen to match for exact string as a part of this PR. Let me know if anything else is preferred.

We could also give weights to certain characteristics like name of block over other ones like its description. This function could be expanded to support that in the future.